lib/pages/wp-admin/wp-admin-jetpack-page.js

```diff
@@ -12,7 +12,10 @@ export default class WPAdminJetpackPage extends AsyncBaseContainer {
 	}
 
 	async connectWordPressCom() {
-		const selector = By.css( 'a.jp-jetpack-connect__button' );
+		// TODO: remove `a.jp-jetpack-connect__button,` after Jetpack 6.9 release
+		const selector = By.css(
+			"a.jp-jetpack-connect__button,.jp-connect-full__button-container a[href*='register']"
+		);
```

Comment:
@brbrr have you tried to fix the problem with the button not always being clickable here? Or was it something else? I wonder if I overcomplicated it here where I've tried to solve the button click issue and remove the `sleep` statement: https://github.com/Automattic/wp-e2e-tests/pull/1739. Can you please take a look when you get a chance? Thanks!

Reply:
No. Jetpack 6.9 will introduce new connection splash screen which introduces new connection button.

Reply:
Thanks for the clarification @brbrr 👍